packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -105,7 +105,7 @@ const AppConfig = ({children, locals = {}}) => {
             // Set 'enablePWAKitPrivateClient' to true to use SLAS private client login flows.
             // Make sure to also enable useSLASPrivateClient in ssr.js when enabling this setting.
             enablePWAKitPrivateClient={false}
-            slasPrivateClientProxyEndpoint={slasPrivateClientProxyEndpoint}
+            privateClientProxyEndpoint={slasPrivateClientProxyEndpoint}
```

Comment:
Aside: how can we detect a similar issue like this (wrong prop name)? Both our unit and e2e tests are not catching it, it looks like.

Reply:
Yeah, this was a miss and it took removing the fallback code before we caught it.

With the fallback in place, it would have taken a specific scenario to see the error - having a SLAS private client proxy set somewhere other than /mobify/slas/private.

The base path remote test environment probably did not catch this since it also was able to handle a request to www.example.com/mobify/proxy/private since the test zone we were using had other rules for handling root requests enabled.